### PR TITLE
Fixed issue : use smooth scrolling for integrated GitHub issue. #396

### PR DESCRIPTION
**Pull request by Chaitanyassr**
https://chaitanyassr.github.io/labs.fossasia.org/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -641,24 +641,27 @@ <h1>How Can I Participate in FOSSASIA projects?</h1>
...
 		</script>
-    </body>
-</html>
+ 
 
 <style>
 html, body {
     max-width: 100%;
     overflow-x: hidden;
+    scroll-behavior: smooth;
```
Comment (Chaitanyassr) [inline index.html]: this line is for smooth scrolling.

**Comment by mariobehling** [comment]
Please squash your commits to make it easier for reviewers.

**Comment by Chaitanyassr** [comment]
> The scrolling conflicts with the existing scrolling feature, e.g. try "go to top" on the footer and you will see the existing scrolling is delayed.

sir, heres the new deployed link - 
https://chaitanyassr.github.io/labs.fossasia.org/

**Comment by Chaitanyassr** [comment]
> The scrolling conflicts with the existing scrolling feature, e.g. try "go to top" on the footer and you will see the existing scrolling is delayed.

sir, here the new deployed link, please reload the page if not work
- https://chaitanyassr.github.io/labs.fossasia.org/

**Comment by Chaitanyassr** [comment]
> Please squash your commits to make it easier for reviewers.

okay Sir.

**Pull request closed without merging**